kratos/tests/cpp_tests/utilities/test_intersection_utilities.cpp

```diff
@@ -584,6 +585,95 @@ namespace Testing {
         KRATOS_CHECK_EQUAL(int_id, 1);
     }
 
+    KRATOS_TEST_CASE_IN_SUITE(ComputeTetrahedraLineIntersection, KratosCoreFastSuite)
```

Comment:
I think this is a pretty large test. I'd suggest separating in smaller ones. This eases a lot (eventual) debugging.

Reply:
This is not done yet.

Reply:
Okay

Reply:
Done